lib/food.js

```diff
@@ -42,10 +42,20 @@ class Food {
                 return new Food(response)
             })
             .then(function(food) {
-                node.append(food.toHTML())
-
+                node.prepend(food.toHTML())
             })
     }
+
+    static validate(food) {
+    let msg = 'Please enter a';
+      if(food.name === '') {
```

Comment:
yup, i'd go with a conditional here for now, then move that to a loop if you have more than just a few fields to validate